mod-data-import-converter-storage-server/src/test/java/org/folio/rest/impl/association/JobToActionProfileTest.java

```diff
@@ -251,21 +484,39 @@ private ActionProfile createActionProfile(TestContext testContext) {
   public void clearTables(TestContext context) {
     Async async = context.async();
     PostgresClient pgClient = PostgresClient.getInstance(vertx, TENANT_ID);
-    pgClient.delete(ASSOCIATION_TABLE_NAME, new Criterion(), associationsDeleteEvent -> {
-      if (associationsDeleteEvent.failed()) {
-        context.fail(associationsDeleteEvent.cause());
-      }
-      pgClient.delete(JOB_PROFILES_TABLE_NAME, new Criterion(), jobProfilesDeleteEvent -> {
-        if (jobProfilesDeleteEvent.failed()) {
-          context.fail(jobProfilesDeleteEvent.cause());
-        }
-        pgClient.delete(ACTION_PROFILES_TABLE_NAME, new Criterion(), actionProfilesDeleteEvent -> {
-          if (actionProfilesDeleteEvent.failed()) {
-            context.fail(actionProfilesDeleteEvent.cause());
-          }
-          async.complete();
+
+    pgClient.delete(ACTION_TO_MATCH_PROFILES, new Criterion(), actionToMatchProfiles -> {
+      stopIfFailed(context, actionToMatchProfiles);
+      pgClient.delete(ACTION_TO_ACTION_PROFILES, new Criterion(), actionToActionProfiles -> {
+        stopIfFailed(context, actionToActionProfiles);
+        pgClient.delete(ACTION_TO_MAPPING_PROFILES, new Criterion(), actionToMappingProfiles -> {
+          stopIfFailed(context, actionToMappingProfiles);
+          pgClient.delete(MATCH_TO_ACTION_PROFILES, new Criterion(), matchToActionProfiles -> {
+            stopIfFailed(context, matchToActionProfiles);
+            pgClient.delete(JOB_TO_MATCH_PROFILES, new Criterion(), jobToMatchProfiles -> {
+              stopIfFailed(context, jobToMatchProfiles);
+              pgClient.delete(JOB_TO_ACTION_PROFILES, new Criterion(), associationsDeleteEvent -> {
+                stopIfFailed(context, associationsDeleteEvent);
+                pgClient.delete(JOB_PROFILES, new Criterion(), jobProfilesDeleteEvent -> {
+                  stopIfFailed(context, jobProfilesDeleteEvent);
+                  pgClient.delete(ACTION_PROFILES, new Criterion(), actionProfilesDeleteEvent -> {
+                    stopIfFailed(context, actionProfilesDeleteEvent);
+                    async.complete();
+                  });
+                });
+              });
+            });
+          });
         });
       });
     });
+
+
+  }
+
+  private void stopIfFailed(TestContext context, AsyncResult<UpdateResult> actionToActionProfiles) {
```

Comment:
Better to rename 'actionToActionProfiles' reference to something more general - asyncResult

Reply:
Done